### PR TITLE
Updating ngsa-typescript documentation

### LINKED ISSUE #269: Update /NewApp/ngsa-typescript
Remove/update references:
- helium => ngsa
- webv => loderunner
- ensure any webv commands are update to work properly running loderunner

- References #261

### PR DESCRIPTION
**Pull request by aflinchb**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- Updating ts docs
- Removing references to helium/webv
- Updated packages 
- Removed duplicate test files

## Validation

- [x] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes #269

### CODE REVIEW

File: src/ngsa-typescript/README.md [hunk 1]
```diff
@@ -42,89 +45,41 @@ npm start
...
 
-# there may be a validation error on the /healthz/ietf endpoint test
-#   json: status: warn : Expected: pass
-# the "warn" status indicates a slower than normal response time
-# this is OK and will occasionally occur
-
-# benchmark.json is a 300 request test that covers the entire API
-
-```
+> TODO: Implement similar coverage tests as csharp
```
Comment (aflinchb) [inline src/ngsa-typescript/README.md:83]: I added a TODO which may be not the point of closing out this task for M1.  If preferred, I can remove the todo and section from the readme for now, and we can add it back when the feature is implemented as part of the bigger typescript app story (to be created).

File: src/ngsa-typescript/README.md [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
-> Build a Node.js and Restify Web API application that calls a data service running on localhost:4122
+## Overview
 
-## Prerequisites
+- This project implements the Web API described in [/src/ngsa-csharp/README.md](https://github.com/retaildevcrews/ngsa/blob/main/src/ngsa-csharp/README.md) in Node.js.
+- It calls the [Ngsa.Dataservice](https://github.com/retaildevcrews/ngsa/tree/main/src/ngsa-csharp/Ngsa.DataService) which either queries an in-memory database or a Cosmos DB instance.
+- [Ngsa.LodeRunner](https://github.com/retaildevcrews/ngsa/tree/main/src/ngsa-csharp/Ngsa.LodeRunner) is used for end-to-end testing and load generation.
+
+## Pre-requisites
```
Comment (bartr) [inline src/ngsa-typescript/README.md:9]: can we change the above links to relative paths like the first one?

File: src/ngsa-typescript/README.md [hunk 3]
```diff
@@ -42,89 +45,41 @@ npm start
...
-#   json: status: warn : Expected: pass
-# the "warn" status indicates a slower than normal response time
-# this is OK and will occasionally occur
-
-# benchmark.json is a 300 request test that covers the entire API
-
-# cd to root of repo
-cd ..
+# longer end-to-end test
+dotnet run -- -s http://localhost:4120 -f benchmark.json
```
Comment (bartr) [inline src/ngsa-typescript/README.md:76]: additionally, we could use docker run

Seems more natural to use docker as some TS devs might not have dotnet installed?

**Review by bartr — APPROVED** [review]
approved with a couple of comments

**Pull request merged**